geomesa-filter/src/main/scala/org/locationtech/geomesa/filter/visitor/QueryPlanFilterVisitor.scala

```diff
@@ -89,6 +89,11 @@ class QueryPlanFilterVisitor(sft: SimpleFeatureType) extends DuplicatingFilterVi
       visitBinarySpatialOp(super.visit(f, data).asInstanceOf[Overlaps], sft)
     }
 
+  override def visit(f: Contains, data: AnyRef): AnyRef =
```

Comment:
As we are handling contains, the order of arguments matters.  Do adding these changes handle the subtlety?